lib/voyager-message/src/module.rs

```diff
@@ -176,12 +176,18 @@ impl ClientModuleInfo {
 
     pub fn ensure_ibc_interface(
         &self,
-        ibc_interface: impl AsRef<str>,
+        expected_interfaces: Vec<impl AsRef<str>>,
```

Comment:
instead of Vec, you can take `impl IntoIterator<Item = impl AsRef<str>>`

Reply:
gotcha